### DISCUSSION TITLE
Docker Restart

### DISCUSSION THREAD

**Discussion by Hassan1995dev** [markshust/docker-magento#586] [Q&A]
i import database and run 

bin/magento setup:install --base-url=http://myproject.test --db-host=db --db-name=richy --db-user=root --db-password=magento --use-rewrites=1 --backend-frontname=admin123

and create host file  , after this my project run  but whenever i restart my system and run http://myproject.test it gives 403 forbidden nginx error.

and i have to again run mentioned above setup install command

**Comment by markshust** [comment]
If you are importing an existing install, there is no need to run `bin/magento setup:install`, as that is for creating new Magento instances.
